src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -549,25 +549,27 @@ def _record_cross_pollination_stats(stats):
   # If no stats were gathered due to a timeout or lack of corpus, return.
   if not stats:
     return
-  # BigQuery not available in local development.This is necessary because the
-  # untrusted runner is in a separate process and can't be easily mocked.
-  if environment.get_value('LOCAL_DEVELOPMENT') or environment.get_value(
-      'PY_UNITTESTS'):
-    return
 
   bigquery_row = {
       'project_qualified_name': stats.project_qualified_name,
       'method': stats.method,
       'sources': stats.sources,
       'tags': stats.tags if stats.tags else '',
       'initial_corpus_size': stats.initial_corpus_size,
-      'corpus_size': stats.minimized_corpus_size_units,
+      'corpus_size': stats.corpus_size,
       'initial_edge_coverage': stats.initial_edge_coverage,
       'edge_coverage': stats.edge_coverage,
-      'initial_feature_coverage': stats.initial_feature_coverate,
+      'initial_feature_coverage': stats.initial_feature_coverage,
       'feature_coverage': stats.feature_coverage
   }
 
+  # BigQuery not available in local development.This is necessary because the
+  # untrusted runner is in a separate process and can't be easily mocked.
+  # Check here instead of earlier to test for attribute errors above.
```

Comment:
Nit: rather than call out the attribute errors specifically, just mention that we return as late as possible to test as much of the function as we can.